econml/inference/_inference.py

```diff
@@ -486,9 +498,13 @@ def intercept__inference(self, T):
             warn("Final model doesn't have a `intercept_stderr_` attribute. "
                  "Only point estimates will be available.")
             intercept_stderr = None
+        # d_t=1 here since we measure the effect across all Ts
+        # Omit treatment name argument to fall back to the default
         return NormalInferenceResults(d_t=1, d_y=self.d_y, pred=self.fitted_models_final[ind].intercept_,
                                       pred_stderr=intercept_stderr,
-                                      inf_type='intercept', **self._input_names)
+                                      inf_type='intercept',
+                                      feature_names=self._input_names["feature_names"],
```

Comment:
I don't like that this is accessing the _input_names dict directly. This hsould instead be accessing the public cate_feature_names and cate_output_names.

Reply:
Fixed!